be/src/olap/rowset/segment_group.cpp

```diff
@@ -697,6 +697,13 @@ const TabletSchema& SegmentGroup::get_tablet_schema() {
     return *_schema;
 }
 
+int SegmentGroup::get_num_zone_map_columns() {
```

Comment:
this function only get the column num of maybe has ZoneMap, in agg, uniq, replace table  itt returns the key columns count, it certainly sorted and had ZoneMap, in duplicate table it return the all column count, the key columns has sorted and has  ZoneMap, the value column in new code also has ZoneMap, but not sorted, so it is not  `Most of the time, you will have to filter the delete predicate row by row.`,  only  when delete by duplicted values, in otherwise if we sorted the duplicate table's value column, it will be the same with key column essentially, so that the value column is unmeaningful